### ISSUE TITLE
Dashboards for DCR for Apps

### ISSUE THREAD

**Issue by jamesgorrie** [guardian/dotcom-rendering#8907]
We'd like to track the metrics of articles served via `dotcom-rendering` vs `apps-rendering`.

It would be good if we could also enrich this data for reasons we aren't serving from DCR based on our release plan: #8750 

The work will probably need to be done in MAPI based on the [`RenderingPlatformPicker`](https://github.com/guardian/mobile-apps-api/blob/main/common-play/src/main/scala/data/RenderingPlatformPicker.scala).

TL;DR; on the release plan, we could filter on:

We will need to support these for beta release:
- Offline reading

We will need to support these for prod release:
- Dark more

Then we can filter on these aspects in prod:
- Content is commentable
- Content is of a specific layout
- Content has video

The last list is what we should report on.

**Comment by rhiannareechaye** [comment]
Closing this ticket as it's a replica of [#8992](https://github.com/guardian/dotcom-rendering/issues/8992)
